jupyterhub/handlers/base.py

```diff
@@ -873,10 +873,14 @@ def abort():
         await user.stop(server_name)
 
     async def stop_single_user(self, user, server_name=''):
+        start_time = time.perf_counter()
         if server_name not in user.spawners:
             raise KeyError("User %s has no such spawner %r", user.name, server_name)
         spawner = user.spawners[server_name]
         if spawner.pending:
+            SERVER_STOP_DURATION_SECONDS.labels(
+                status=ServerStopStatus.pending
+            ).observe(time.perf_counter() - start_time)
```

Comment:
Let's remove this pending record. It doesn't really make sense to record the time it takes to not do anything (the server is not stopping here, the request is being cancelled because a stop is already pending)